calculator/src/components/button.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react'
```

Comment:
It looks like an old duplicate file. Remember to remove dead code as well as dead files before submitting your work.